### PR TITLE
Jormun: refactor admin marshall from bragi

### PR DESCRIPTION
**Pull request by pbougue**
* use city object from bragi, not administrative_region sub-object (can be empty)
* return str for lat,lon as in kraken

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/Places.py [hunk 1]
```diff
@@ -122,13 +123,13 @@ def create_administrative_regions_field(geocoding):
 
 
 def get_lon_lat(obj):
-    if not obj:
+    if not obj or not obj.get('geometry') or not obj.get('geometry').get('coordinates'):
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/interfaces/v1/Places.py:126]: I don't see the point of this check

File: source/jormungandr/jormungandr/interfaces/v1/Places.py [hunk 2]
```diff
@@ -142,8 +143,25 @@ class AdministrativeRegionField(fields.Raw):
...
+            "level":
+                int(geocoding.get('level')) if geocoding.get('level') else None ,
+            "name": geocoding.get('name'),
+            "label": geocoding.get('label'),
+            "id": geocoding.get('id'),
+            "coord": {
+                "lat": lat,
+                "lon": lon
+            },
+            "zip_code": geocoding.get('postcode'),
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/interfaces/v1/Places.py:161]: shouldn't we `format_zip_code` here ?

**Review by antoine-de — APPROVED** [review]

File: source/jormungandr/jormungandr/interfaces/v1/Places.py (hunk 1 above)
Comment (pbougue) [reply source/jormungandr/jormungandr/interfaces/v1/Places.py:126]: just (over-)protecting code bellow

File: source/jormungandr/jormungandr/interfaces/v1/Places.py (hunk 2 above)
Comment (pbougue) [reply source/jormungandr/jormungandr/interfaces/v1/Places.py:161]: in geocodejson it's a string not a table

**Pull request merged**